bwakit/Makefile

```diff
@@ -7,7 +7,8 @@ build: ${build_tool}
 
 ${build_tool}: Dockerfile
 	docker build -t ${name}:${tag} .
-	docker tag -f ${name}:${tag} ${name}:latest
+	-docker rmi ${name}:latest
```

Comment:
I don't think you need the rmi on latest, doing `docker tag ${name}:${tag} ${name}:latest` should move the tag from the "old" latest image to the new `${name}:${tag}` image.